lib/vanagon/platform/windows.rb

```diff
@@ -289,6 +289,105 @@ def output_dir(target_repo = "")
         File.join("windows", target_repo, @architecture)
       end
 
+      # Generate correctly formatted wix elements that match the
+      # structure of the directory input
+      #
+      # @param services, Array of components services
+      # and optionally:
+      # @return [string] correctly formatted wix element string
+      def generate_wix_dirs(services)
+        directories = []
+        services.map { |svc| directories.push({ :path => svc.service_file, :id => "#{svc.id}BINDIR" }) }
```

Comment:
For example, if both mco and puppet had the same bin dir, at the point in the wix file where we have the bin directory it will look like: 

`<Directory Name="bin" Id="bin">
    <Directory Id="MARIONETTECOLLECTIVEBINDIR" />
    <Directory Id="PUPPETBINDIR" />
  </Directory>`